indra/newview/installers/windows/installer_template.nsi

```diff
@@ -1,902 +1,902 @@
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;

```

Comment:
Was this commited by a mistake? I don't see any changes in this files sans mac's line endings to window's instalator file.

Reply:
You're right: when I checked "Hide whitespace," GitHub says there are only whitespace differences.

Does NSIS actually care any more? I realize our git-hooks coding policy wants Windows line endings on any file with a relative path including "/windows/", but does it matter to the program that reads that file?

Reply:
Likely doesn't care, but keeping it this way can be usefull just to prevent 'line ending batteles' due to people opening the thing in different editors.

Reverted this specific file to unlock builds and to prevent merge conflicts (I think we did some changes in other branches to this file)

Reply:
Thank you.